### PR TITLE
add how-to section for usage

### PR DESCRIPTION
**Pull request by nginx-jack**
This PR adds a basic how-to section.

### CODE REVIEW

**Review by ADubhlaoich — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -6,6 +6,23 @@ The action pushes to an Azure Storage Container, and assumed an existing reverse
...
+1. Head to the actions section of your repository, for example https://github.com/yourOrg/yourRepo/actions
+
+1. On the left side, click on Build and deploy (docs)
+![Build and deploy (docs)](/images/build-and-deploy.png "Build and deploy (docs)")
+1. On the right side, you should now see a Run workflow button
+![Run Workflow](/images/run-workflow.png "Run Workflow")
+1. Clicking this will open a menu which lets you select the branch you want to build, and which environment you'd like to deploy to
+1. Preview will generate a custom url for any branch you want, and will be available in the workflow output.
+All builds, regardless of environment, will print a url to the summary.
+![Summary](/images/summary.png "Summary")
```
Comment (mjang) [inline README.md:20]: @nginx-jack I'm overdoing this a bit, as I'm practicing my knowledge of our writing style guides (which you're not required to follow)
- https://nginxsoftware.atlassian.net/wiki/spaces/ENGUSEMEA/pages/478053389/NGINX+Documentation+Style+Guide
- https://nginxsoftware.atlassian.net/wiki/spaces/~62c7dc4b40923988ba0b4dfe/pages/2576515248/Concatenated+NGINX+Documentation+Style+Guide (@ADubhlaoich 's WIP)

Question: what happens if someone doesn't have a "Build and deploy (docs)" action? (fill in the \<docops\> blank on my suggested response)

```suggestion
## How-to

These instructions apply only to NGINX GitHub doc repositories.

1. Navigate to the actions section of your repository, for example https://github.com/yourOrg/yourRepo/actions

1. On the left side of the page, select Build and deploy (docs). If you don't see this option, contact <doc-ops>.
![Build and deploy (docs)](/images/build-and-deploy.png "Build and deploy (docs)")
1. On the right side, you will see a Run workflow button.
![Run Workflow](/images/run-workflow.png "Run Workflow")
1. Select Run workflow. This opens a menu which lets you select the branch you want to build, and which environment you'd like to deploy to
1. The preview environment generates a custom URL for your preferred branch, and will be available in the workflow output.
All builds, regardless of environment, prints an URL to the summary.
![Summary](/images/summary.png "Summary")
```

File: README.md [hunk 2]
```diff
@@ -6,6 +6,23 @@ The action pushes to an Azure Storage Container, and assumed an existing reverse
...
+
+Preview URLs wil be added as a comment to PRs automatically.
+
+_note: autodeploy from branch is currently in progress. This docs will be updated when available._
```
Comment (mjang) [inline README.md:24]:
```suggestion
The action automatically adds the Preview URL to the PR automatically, as a comment.

_note: autodeploy from branch is currently in progress. This docs will be updated when available._
```

I'm not sure why you have this last line. (line 24)

**Review by mjang — APPROVED** [review]
@nginx-jack Your How-to is easy to follow. I do have suggestions.

File: README.md (hunk 1 above)
Comment (nginx-jack) [reply README.md:20]: Thanks @mjang , this looks much better!
You, or someone from docsops, would need to have followed the `caller example` below this, in order for you to see this action.
Someone from docOps should be setting this up for each repository that needs it, so reaching out to us directly is probably the best course of action if you don't have the action.

**Review by danielledeleo — APPROVED** [review]
Thank you @mjang for the copyediting. Looks great.

**Pull request merged**